.travis.yml

```diff
@@ -3,5 +3,6 @@ sudo: false
 language: scala
 jdk:
   - openjdk6
+  - oraclejdk8
```

Comment:
This runs tests on Java 6 and Java 8. In the build file it checks the current version of Java and adds scala 2.12.0-MX versions to `crossScalaVersions` only if it detects that it is running on Java 8.